### DISCUSSION TITLE
Custom CSS

### DISCUSSION THREAD

**Discussion by xmha97** [cefsharp/CefSharp#3950] [General]
I just want to know if I can use Custom CSS like OBS Studio?

![image](https://user-images.githubusercontent.com/23485114/148620358-c10f3c82-abf3-4213-bc02-6b5b17672d2c.png)

OBS Studio also uses [CEF](https://bitbucket.org/chromiumembedded/cef/src/master/README.md) (libcef.dll not CefSharp)

**Comment by amaitland** [comment]
In future please use discussions to ask questions. https://github.com/cefsharp/CefSharp/discussions

You can use JavaScript to modify the DOM, including injecting CSS.

https://github.com/cefsharp/CefSharp/wiki/General-Usage#1-how-do-you-call-a-javascript-method-from-net
